ipatests/prci_definitions/nightly_latest_pki.yaml

```diff
@@ -791,11 +795,24 @@ jobs:
       args:
         build_url: '{pki-fedora/build_url}'
         update_packages: True
-        test_suite: test_integration/test_ipahealthcheck.py
+        test_suite: test_integration/test_ipahealthcheck.py::TestIpaHealthCheck test_integration/test_ipahealthcheck.py::TestIpaHealthCheckWithoutDNS
         template: *pki-master-latest
         timeout: 4800
         topology: *master_1repl
 
+  pki-fedora/test_ipahealthcheck_adtrust:
```

Comment:
I don't think it makes sense to add the new test to pki testing. You can keep the modification for pki-fedora/test_ipahealthcheck (run only TestIpaHealthCheck and TestIpaHealthCheckWithoutDNS) and remove this one.

Reply:
@flo-renaud, removed the test related to trust from pki testing